mock-backend/src/utils/categoryFilter.js

```diff
@@ -0,0 +1,10 @@
+const categoryFilter = (category, productsList) => {
+  if (category === "") {
+    return productsList;
+  }
+  return productsList?.filter((product) => {
+    return product.tags.includes(category);
+  });
+};
```

Comment:
Also in general in can write simplier like
```
const categoryFilter = (category, productsList) => !category ? productsList : productsList?.filter(product => product.tags.includes(category))

```

What do you think ?

Reply:
@mhevyk I had to revert some of the changes you suggested. The reason is that the backend expects to receive tags param always. if we don't  provide at least empty string, we get nothing. Empty string gets us all products, without category filtering.

Reply:
Done